fe/fe-core/src/main/java/com/starrocks/server/GlobalStateMgr.java

```diff
@@ -1629,10 +1633,14 @@ public void replayJournal(long toJournalId) throws JournalException {
             cursor = journal.read(startJournalId, toJournalId);
             replayJournalInner(cursor, false);
         } catch (InterruptedException | JournalInconsistentException e) {
-            LOG.warn("got interrupt exception or inconsistent exception when replay journal, will exit, ", e);
+            LOG.warn("got interrupt exception or inconsistent exception when replay journal {}, will exit, ",
+                    replayedJournalId.get(),
```

Comment:
Throwable should be catch here and print the failed journalId.

Reply:
should be replayedJournalId.get() + 1? 
=====
Good point. Will fix it.

Throwable should be catch here and print the failed journalId.
=====
It is printed in replayJournalInner() because replayer, checkpointer, and replay procedure when startup will all call replayJournalInner().